src/components/TodoList/TodoList.tsx

```diff
@@ -25,198 +35,49 @@ export const TodoList: React.FC = () => {
         </thead>
 
         <tbody>
-          <tr data-cy="todo">
-            <td className="is-vcentered">1</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">delectus aut autem</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
+        {todos.map(todo => (
+          <tr data-cy="todo" className="" key={todo.id}>
+            <td className="is-vcentered">{todo.id}</td>
+            {todo.completed ? (
+              <td className="is-vcentered">
+                <span className="icon" data-cy="iconCompleted">
+                  <i className="fas fa-check" />
                 </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">2</td>
-            <td className="is-vcentered"> </td>
-
+              </td>
+            ) : (
+              <td className="is-vcentered" />
+            )}
             <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">
-                quis ut nam facilis et officia qui
+              <p
+                className={classNames({
+                  'has-text-danger': todo.completed === false,
+                  'has-text-success': todo.completed === true,
+                })}
```

Comment:
do we need this comparison?
```suggestion
                  'has-text-danger': !todo.completed,
                  'has-text-success': todo.completed,
                })}
```